### ISSUE TITLE
[KNX2] None of the configured GAs on channel could handle the command

### ISSUE THREAD

**Issue by rsterz** [openhab/openhab2-addons#3641]
My issue is that part of my rules don't fire to GAs unless the related things and items look the same. I am using these rules with my old OH1 instance having no issues there.

## Expected Behavior
Rules should trigger writing to a GA using number values while using MyItem.sendCommand(value). I tried converting the number value to string but that doesn't change the behavior. Using type number instead of number-control doesn't change the behavior as well.

## Current Behavior
I receive the following error message for several rules:
```
2018-06-09 00:05:10.214 [TRACE] [.internal.handler.DeviceThingHandler] - Handling command '2' for channel 'knx:device:bridge:generic:hzSWVBuero'
2018-06-09 00:05:10.216 [DEBUG] [.internal.handler.DeviceThingHandler] - None of the configured GAs on channel 'knx:device:bridge:generic:hzSWVBuero' could handle the command '2' of type 'DecimalType'
```

For one rule that is working I receive the following output:
```
2018-06-09 00:07:31.691 [TRACE] [.internal.handler.DeviceThingHandler] - Handling command '3' for channel 'knx:device:bridge:generic:valloxStatus'
2018-06-09 00:07:31.694 [DEBUG] [nx.internal.client.AbstractKNXClient] - Wrote value '3' to datapoint 'command DP 11/4/0 'knx:ip:bridge', DPT id 6.010, low priority' (0. attempt).

```
## Possible Solution
I have no clue why these two items don't result in the same behavior.

## Steps to Reproduce (for Bugs)
Things configuration (excerpt):
```
    Thing device generic "Generic" {
        Type number-control : hzSWVBuero    "Verstellung Büro" [ ga="6.010:9/5/3+6.010:9/6/3" ]
        Type number-control : valloxStatus  "Profil"           [ ga="6.010:11/4/0" ]
     }
```

Items configuration (excerpt):
```
Number HZ_SV_BU "Verstellung Büro [%d]" <temperature> (HZ_SV) { channel="knx:device:bridge:generic:hzSWVBuero" }
Number sendVallox "Lüftungsprofil [%d]" <climate> { channel="knx:device:bridge:generic:valloxStatus" }

```
Rules configuration (excerpt):
```
rule "HZ_ZW_BU changed"
when
	Item HZ_ZW_BU changed
then
	counter = BU_counter
	BU_counter = (HZ_ZW_BU.state as Number - 22) * 2
	if (BU_counter != counter) {
		HZ_SV_BU.sendCommand(BU_counter.intValue().toString())
		logInfo("tempinfo",BU_counter.intValue().toString())
	}
end

rule "ValloxToKNX"
when
    Item stateVallox changed
then
    val DecimalFormat df = new DecimalFormat("#")
    val Number ValloxRounded = (df.format(stateVallox.state as DecimalType))
    if (sendVallox.state != stateVallox.state) sendVallox.sendCommand(ValloxRounded.toString())
end
```

## Context
The first rule is used to calculate and send the shift value required to change my room temperature. This is the way that Busch Jaeger Prion RTR communicate with each other.
The second rule is used to send the current state of the ventilation system to the KNX bus to my Busch Jaeger panel.

## Your Environment
* Version used: OpenHAB 2.3.0 (apt package)
* Environment name and version (e.g. Chrome 39, node.js 5.4, Java 8, ...): Chrome 67, Java 8
* Operating System and version (desktop or mobile, Windows 10, Raspbian Jessie, ...): Raspbian Stretch (Pi3B+)

**Comment by kaikreuzer** [comment]
> Rules should trigger writing to a GA using number values while using MyItem.sendCommand(value)

If this is your expectation, why do you use a [control channel type](https://www.openhab.org/addons/bindings/knx/#control-channel-types) where "where the KNX bus does not own the physical state"? Those mean that you expect only "inward" commands, i.e. from KNX to openHAB as KNX anyhow wouldn't be allowed to react on a command from openHAB.
For control channels, openHAB sends out state updates to the KNX dps, so that those can synchronise themselves with the "official" state that is within openHAB. (/cc @sjka correct me if anything is not exactly right).

There is therefore imho no bug here - if you have further questions about it, I'd suggest to discuss them on https://community.openhab.org - thanks!

**Comment by rsterz** [comment]
Thanks Kai for the explanation. Maybe it was a little bit late when I opened this thread last night.

You are doing great work on the KNX2 addon. I am happy that I can switch from the old plugin soon to this one. This is my last challenge with the addon before switching over finally.

As I described I changed the Type to number as well without a changing result. The binding tells me that it cannot handle a number for the given channel - which should be able to accept it as it is a DPT 6.010. 
From my point of view the binding is reacting differently for two similar sendCommand calls (I have one that is working and 10 not working - examples were in my initial post). That's what I didn't understand and see it as a misbehavior which I couldn't resolve.

I will move the discussion to the community portal as I am still looking for a solution sending a number to a GA - which should not be that difficult.
